### PR TITLE
Models

### PR DESCRIPTION
**Pull request by sbsmith86**
#### What's this PR do?
* Add `.editorconfig` file.
* Moves models into `Rogue/Models` directory/namespace.
* Adds models for each table.

#### Relevant tickets
Addresses #1

### CODE REVIEW

**Comment by DFurnes** [comment]
No diff since it was only renamed, but the User model class is still in the root `\Rogue` namespace (rather than `\Rogue\Models`).

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     },
     "autoload": {
         "classmap": [
-            "database"
+            "database",
+            "app/Models"
```
Comment (DFurnes) [inline composer.json]: This shouldn't be necessary since the models are all loaded in the PSR-4 `Rogue\` namespace defined below.

**Comment by sbsmith86** [comment]
@DFurnes ahhh yep. good catch! Wasn't ready to really deal with users yet so I didn't even look at this file haha.

**Comment by DFurnes** [comment]
👍 ⛵

**Pull request merged**